build_and_push_custom_image_to_aws_ecr.sh

```diff
@@ -0,0 +1,16 @@
+echo "Building and push The TIBCO DeveloperHub custom docker image to AWS ECR"
```

Comment:
@sgantaya should we add a README for this file and folder?

Reply:
@hpeters83 Can you please answer this, should we add README .

Reply:
No this is just an example script (like we have example templates)